sdk/containerregistry/Azure.Containers.ContainerRegistry/tests/ContainerRepositoryClientLiveTests.cs

```diff
@@ -0,0 +1,70 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Threading.Tasks;
+using Azure.Core.TestFramework;
+using NUnit.Framework;
+
+namespace Azure.Containers.ContainerRegistry.Tests
+{
+    public class ContainerRepositoryClientLiveTests : RecordedTestBase<ContainerRegistryTestEnvironment>
+    {
+        private readonly string _repositoryName = "library/hello-world";
+
+        public ContainerRepositoryClientLiveTests(bool isAsync) : base(isAsync)
+        {
+        }
+
+        private ContainerRepositoryClient CreateClient()
```

Comment:
nit: If the client creation is not differentiated per test and will always be required, it may make sense to mark this as a `[SetUp]` method and then you can remove the need to call it at the beginning of each test.

Reply:
Ah, I did not know this, thanks!